source/draw/gpu/opengl/GraphicBuffer.ooc

```diff
@@ -101,6 +100,10 @@ GraphicBuffer: class {
 		This _alignedWidth = Int[count] new()
 		memcpy(This _alignedWidth data, alignedWidth, count * Int size)
 	}
+	kean_draw_graphicBuffer_new: unmangled static func (backend, nativeBuffer, handle: Pointer, size: IntVector2D, pixelStride: Int,
+			format: GraphicBufferFormat, ownsMemory: Bool) -> This {
```

Comment:
I'm not sure about formatting here, but maybe @marcusnaslund has other idea for this.